tests/cli/test_boot_live-iso.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
```

Comment:
Could be a good idea to actually mention that this needs to be executed from inside the VM. 

Or even better group the existing files in different directories to indicate where they will be executed.